app/assets/javascripts/banners.js.coffee

```diff
@@ -23,3 +31,21 @@ App.Banners =
     $("#banner_image").on
```

Comment:
I would remove all code related to banner images, as they won't be used anymore (also from the Banners ddbb table).